### PR TITLE
to support eventio 1.5 update dtype of true_image

### PR DESCRIPTION
**Pull request by kosack**
all CI pipelines currently fail since eventio1.5 now stores the true_image field as a int instead of a float.  This should fix it, and require eventio>=1.5

### CODE REVIEW

**Review by maxnoe — DISMISSED** [review]

File: setup.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
     install_requires=[
         "astropy>=4,<5",
         "bokeh~=1.0",
-        "eventio>=1.1.1,<2.0.0a0",  # at least 1.1.1, but not 2
+        "eventio>=1.5.0,<2.0.0a0",  # at least 1.1.1, but not 2
```
Comment (HealthyPear) [inline setup.py:47]: at least 1.5.0

**Review by HealthyPear — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (maxnoe) [reply setup.py:47]: What's your point?

Comment (maxnoe) [reply setup.py:47]: Please just remove the comment. It does not any further information.

**Comment by kosack** [comment]
it seems opening prod2 files, I still get float32 for true_images for some reason.  @maxnoe do you have an idea why it would be different for different files?

**Comment by HealthyPear** [comment]
Failing tests

```
FAILED ctapipe/io/tests/test_dl1eventsource.py::test_is_compatible - Assertio...
FAILED ctapipe/io/tests/test_dl1eventsource.py::test_metadata - KeyError: "At...
FAILED ctapipe/io/tests/test_dl1eventsource.py::test_subarray - KeyError: "At...
FAILED ctapipe/io/tests/test_dl1eventsource.py::test_max_events - KeyError: "...
FAILED ctapipe/io/tests/test_dl1eventsource.py::test_allowed_tels - KeyError:...
FAILED ctapipe/io/tests/test_dl1eventsource.py::test_simulation_info - KeyErr...
FAILED ctapipe/io/tests/test_dl1eventsource.py::test_dl1_data - KeyError: "At...
FAILED ctapipe/io/tests/test_dl1eventsource.py::test_pointing - KeyError: "At...
FAILED ctapipe/tools/tests/test_tools.py::test_merge - assert 1 == 0
FAILED ctapipe/tools/tests/test_tools.py::test_stage_1_dl1 - assert 1 == 0
FAILED ctapipe/tools/tests/test_tools.py::test_display_dl1 - assert 1 == 0
```

**Comment by kosack** [comment]
Nevermind, I found it - if no image exists, the default in SimTelEventSource was to write 0s as float32s.

**Comment by maxnoe** [comment]
@kosack This is to copy the behaviour of pyhessio, as we wanted to be a drop-in replacement for it. But now it should be better to just put None.

**Review by maxnoe — APPROVED** [review]

**Comment by HealthyPear** [comment]
Still 1 failing test, I report here the output from the CI,

`FAILED ctapipe/tools/tests/test_tools.py::test_display_integrator`

```python
___________________________ test_display_integrator ____________________________

tmpdir = local('/tmp/pytest-of-runner/pytest-0/test_display_integrator0')

    def test_display_integrator(tmpdir):
        from ctapipe.tools.display_integrator import DisplayIntegrator
    
        mpl.use("Agg")
    
>       assert (
            run_tool(
                DisplayIntegrator(),
                argv=shlex.split(f"--f={GAMMA_TEST_LARGE} " "--max_events=1 "),
                cwd=tmpdir,
            )
            == 0
        )
E       assert 1 == 0
E         +1
E         -0

ctapipe/tools/tests/test_tools.py:518: AssertionError
```

**Comment by kosack** [comment]
> Still 1 failing test, I report here the output from the CI,

Seems that test was using a file that doesn't have true_image, but where it was trying to plot it... Should be an easy fix

**Review by HealthyPear — APPROVED** [review]

**Review by maxnoe — APPROVED** [review]

**Pull request merged**